### DISCUSSION TITLE
Hide email from new users by default for privacy reasons

### DISCUSSION THREAD

**Discussion by SilasPeters** [outline/outline#7309] [Self Hosting]
Hi there! I successfully self-hosted Outline through a docker compose file, thanks for the awesome work!

My problem now is that I host outline for many people, who can land on our page by pressing a single button, thus logging in and revealing their email without them realising that others can see their email address, for the email address is their default username.

For privacy reasons, I would like to prevent that. Can I let Outline set the username to some other field in OIDC, or is it possible to strip the `@hotmail` part of the email adress and add a random amount of numbers after the name?

**Comment by PoolloverNathan** [comment]
Bump
